packages/tools/devtools/devtools-core/src/data-visualization/DataVisualization.ts

```diff
@@ -94,6 +95,23 @@ export interface SharedObjectVisualizers {
 	[k: SharedObjectType]: VisualizeSharedObject;
 }
 
+/**
+ * Specifies renderers for different {@link @fluidframework/shared-object-base#ISharedObject} types.
+ *
+ * @remarks
+ *
+ * - `key`: The type of Shared object ({@link @fluidframework/datastore-definitions#IChannelFactory.Type}).
+ *
+ * - `value`: A renderer that takes a {@link @fluidframework/shared-object-base#ISharedObject} of the
```

Comment:
Looks like this was copy-pasted. We should update this to be in terms of editing.